### PR TITLE
packages: prepare graphql v5.1.0 

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
Describe the purpose of the pull request.

Fixes https://github.com/zino-hofmann/graphql-flutter/issues/1090

In addition, this PR includes the commands to make the changelog with only one command

### CODE REVIEW

**Comment by vincenzopalazzo** [comment]
Mh I think this will need two separate PRs!

**Comment by vincenzopalazzo** [comment]
added the changelog summary from the beta to the official one unfortunately for this time we missed a lot of changelog entry due the bug in the prev changelog generator

File: packages/graphql_flutter/pubspec.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 name: graphql_flutter
 description: A GraphQL client for Flutter, bringing all the features from a modern GraphQL client to one easy to use package.
-version: 5.0.2-beta.7
+version: 5.1.0
 homepage: https://github.com/zino-app/graphql-flutter/tree/master/packages/graphql_flutter
 
 dependencies:
-  graphql: ^5.0.2-beta.6
+  graphql: ^5.0.1
```
Comment (budde377) [inline packages/graphql_flutter/pubspec.yaml:7]: Do you think I makes sense to release graphql before graphql_flutter, so we can get the right version here?

Comment (vincenzopalazzo) [reply packages/graphql_flutter/pubspec.yaml:7]: yes it can have sense, more sense IMHO that put it in a single release

Comment (vincenzopalazzo) [reply packages/graphql_flutter/pubspec.yaml:7]: if you agree I will split in two this PR and release the graphql before

Comment (budde377) [reply packages/graphql_flutter/pubspec.yaml:7]: That sounds great!

Comment (vincenzopalazzo) [reply packages/graphql_flutter/pubspec.yaml]: Ok graphq PR is ready

File: packages/graphql_flutter/changelog.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "package_name": "graphql_flutter",
-  "version": "v5.0.2-beta.7",
```
Comment (budde377) [inline packages/graphql_flutter/changelog.json:3]: Should this be part of this PR?

**Review by budde377 — APPROVED** [review]
Just flagged a simple thing. If it doesn't matter, please ignore and release.

File: packages/graphql_flutter/changelog.json (hunk 2 above)
Comment (vincenzopalazzo) [reply packages/graphql_flutter/changelog.json:3]: Mh! I think yes, the update file is in the other PR :) I just split the the changes in two

**Pull request merged**